test/functional/p2p_filter.py

```diff
@@ -51,6 +56,21 @@ def on_merkleblock(self, message):
     def on_tx(self, message):
         self.tx_received = True
 
+class FilterNodeFRelay(FilterNode):
+    def peer_connect(self, *args, services=NODE_NETWORK|NODE_WITNESS, send_version=True, **kwargs):
```

Comment:
I think you can remove the `services` and `send_version` arguments, and just set them directly below (this function is only ever called once, and you know what `services` and `send_version` should be set to.